editoast/src/views/infra/auto_fixes.rs

```diff
@@ -765,4 +792,47 @@ mod test {
             obj_type: ObjectType::BufferStop,
         })));
     }
+
+    #[rstest::rstest(endpoint, bs_position,
+        case(&Endpoint::Begin, 0.),
+        case(&Endpoint::End, 100.),
+    )]
+    fn missing_track_extremity_buffer_stop_fix(endpoint: &Endpoint, bs_position: f64) {
+        let mut cache = InfraCache::default();
+        let track = TrackSectionCache {
+            obj_id: "track_id".into(),
+            line_code: None,
+            length: 100.,
+            slopes: Vec::new(),
+            curves: Vec::new(),
+            bbox_geo: BoundingBox {
+                0: (0., 0.),
+                1: (0., 0.),
+            },
+            bbox_sch: BoundingBox {
+                0: (0., 0.),
+                1: (0., 0.),
+            },
+        };
```

Comment:
minor: you can probably shorten a bit
```suggestion
        let track = TrackSectionCache {
            obj_id: "track_id".into(),
            length: 100.,
            bbox_geo: BoundingBox {
                0: (0., 0.),
                1: (0., 0.),
            },
            bbox_sch: BoundingBox {
                0: (0., 0.),
                1: (0., 0.),
            },
            ..Default::default()
        };
```